src/graphql-global-id/README.md

```diff
@@ -50,7 +50,11 @@ const fields = {
 };
 ```
 
-# Restoring original ID
+# `toGlobalId`
+
+TKTK
```

Comment:
what's this? 😄 

<img width="717" alt="Screenshot 2020-03-31 17 44 47" src="https://user-images.githubusercontent.com/2174629/78046346-573adf80-7377-11ea-8b67-d96d9a780ed6.png">

Reply:
- https://en.wikipedia.org/wiki/To_come_(publishing)
- https://www.urbandictionary.com/define.php?term=TKTK